code/modules/clothing/glasses/powered.dm

```diff
@@ -12,9 +12,14 @@
 
 /obj/item/clothing/glasses/powered/Initialize()
 	. = ..()
+	START_PROCESSING(SSobj, src)
 	if(!cell && suitable_cell)
```

Comment:
Do they really need to process at all times? I think it's only used for power consumption, so it makes more sense for them to process only when powered.

Reply:
^
Probably should move that to toggle proc